cmd/root.go

```diff
@@ -170,6 +171,7 @@ func bindEnv(flags ...string) bindFunc {
 		}
 		viper.AutomaticEnv()       // read in environment variables for FUNC_<flag>
 		viper.SetEnvPrefix("func") // ensure that all have the prefix
+		viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
```

Comment:
And this duplicate directive is another reason we have a flag refactor/cleanup in the works